src/components/button/button-large/button-large-primary/button-large-primary.styles.ts

```diff
@@ -4,7 +4,7 @@ import { useButtonLargeSharedStyleConfig } from '../button-large.styles';
 export const useButtonLargePrimaryStyleConfig = createUseStylesConfig(({ colors }) => ({
   ...useButtonLargeSharedStyleConfig(),
   activeColorConfig: {
-    titleColor: colors.white,
+    titleColor: colors.black,
```

Comment:
this broke a lot of buttons 
![image](https://user-images.githubusercontent.com/10281464/143912508-983d65e1-00eb-45f5-9942-6265279a84e7.png)